Libplanet.Action.Tests/Mocks/MockAccount.cs

```diff
@@ -0,0 +1,121 @@
+using System.Numerics;
+using Libplanet.Action.State;
+using Libplanet.Crypto;
+using Libplanet.Store.Trie;
+using Libplanet.Types.Assets;
+using Libplanet.Types.Tx;
+
+namespace Libplanet.Action.Tests.Mocks
+{
+    public static class MockAccount
+    {
+        public static IAccount Empty { get; } =
+            new Account(default, new MerkleTrie(new MemoryKeyValueStore()));
+
+        public static IAccount SetBalance(
+            this IAccount account,
+            Address address,
+            FungibleAssetValue amount) =>
+            SetBalance(account, (address, amount.Currency), amount);
+
+        public static IAccount SetBalance(
+            this IAccount account,
+            Address address,
+            Currency currency,
+            BigInteger rawAmount) =>
+            SetBalance(
+                account,
+                (address, currency),
+                FungibleAssetValue.FromRawValue(currency, rawAmount));
+
+        public static IAccount SetBalance(
+            this IAccount account,
+            (Address Address, Currency Currency) pair,
+            FungibleAssetValue amount)
+        {
+            var context = CreateContext(account, pair.Address);
+            var currentAmount = account.GetBalance(pair.Address, pair.Currency);
+            var balanceDelta = currentAmount - amount;
+            return balanceDelta.Sign switch
+            {
+                0 => account,
+                > 0 => account.BurnAsset(context, pair.Address, balanceDelta),
+                < 0 => account.MintAsset(context, pair.Address, balanceDelta.Abs()),
+            };
```

Comment:
Similar as above. Fixture setting should be agnostic to logic as much as possible. This can throw an `Exception` from `IAccount`'s implementation.

Reply:
Hmm... but if the `IAccount` implementation was correct, it would be fine I guess...? Can you explain some kind of `Exception` thrown case if `IAccount.BurnAsset` and `IAccount.MintAsset` are implemented as spec?